notebooks/recommender-system.ipynb

```diff
@@ -0,0 +1,843 @@
+{
```

Comment:
<p>I <strong>love</strong> this application. I think its shaping up to be an awesome and high impact tutorial. </p><p> </p><p>Check out this similar [RealPython](</p><p>https://realpython.com/build-recommendation-engine-collaborative-filtering/#when-can-collaborative-filtering-be-used) tutorial on collaborative filtering. One of the missing pieces at the end is demonstrating effectiveness of the approach. It might even help to run through a step-by-step demo on a 3-by-3 <code>rankings</code>  array so the user sees how the reshapes use the original data to predict new data. </p><p> </p><p>tech document for the most part use 2nd person e.g. "In this tutorial, <em>you</em> will use NumPy..."\</p><p> </p><p>in the <strong>you'll learn</strong>, you should be able to remove the <em>you'll</em> in each bullet</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cooperrc/uconn_numpy_tutorials/pull/11/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>